internal/servers/options_test.go

```diff
@@ -108,6 +127,16 @@ func Test_GetOpts(t *testing.T) {
 		opts := getOpts(WithNonce(nonce))
 		testOpts = getDefaultOptions()
 		testOpts.withNonce = nonce
+		opts.withNewIdFunc = nil
+		testOpts.withNewIdFunc = nil
 		assert.Equal(opts, testOpts)
 	})
+	t.Run("WithNewIdFunc", func(t *testing.T) {
+		opts := getOpts(WithUpdateTags(true))
```

Comment:
It looks like this test might not have been implemented for the new Id func

Reply:
oops!  ty!